cmd/credentialUtil.go

```diff
@@ -114,6 +114,17 @@ func GetOAuthTokenManagerInstance() (*common.UserOAuthTokenManager, error) {
 
 		case "DEVICE":
 			lca.identity = false
+		
+		case "AZCLI":
+			lca.identity = false
```

Comment:
in the future we might be able to improve this by creating an enum for oauth cred type instead of having a bunch of booleans